### PR TITLE
cleanup pass one on en-US lines 1-500

### PR DESCRIPTION
**Pull request by kaiyzen**
we need to do multiple passes on cleaning up the English strings, this is first pass on the initial lines 1-500 in the json string file

### CODE REVIEW

File: src/language/en-US.json [hunk 1]
```diff
@@ -314,7 +315,7 @@
   "switch_point": "Switch Node",
   "current_point": "Current Node",
   "point_null_error": "Node null error",
-  "please_enter_your_wallet_password_to_ensure_your_own_operation_and_keep_your_wallet_safe": "Please enter your wallet password to ensure your own operation and keep your wallet safe.",
+  "please_enter_your_wallet_password_to_ensure_your_own_operation_and_keep_your_wallet_safe": "Please enter your wallet password to ensure it is protected kept safe.",
```
Comment (decentraliser) [inline src/language/en-US.json]: This one needs update

Comment (decentraliser) [reply src/language/en-US.json]: Remove "protected" I guess

File: src/language/en-US.json [hunk 2]
```diff
@@ -326,14 +327,14 @@
...
-  "Please_back_up_the_Keystore_properly_If_your_phone_is_lost_stolen_or_damaged": "Please back up the Keystore properly. If your device is lost, stolen or damaged, Keystore+ password will be able to recover your assets.",
+  "obtaining_a_Keystore_password_is_equal_to_owning_a_wallet_asset": "Possessing a Keystore + password is equal to owning an account/wallet",
+  "Please_back_up_the_Keystore_properly_If_your_phone_is_lost_stolen_or_damaged": "Please back up the Keystore properly. If your device is lost, stolen or damaged, the Keystore + password will be able to recover your accounts/wallets.",
   "offline_storage": "Offline storage",
-  "keep_it_in_a_safe_place_on_the_isolated_network": "Keep it in a safe place on the isolated network. Do not share and store mnemonics in a networked environment, such as emails, photo albums, social network, message, etc.",
+  "keep_it_in_a_safe_place_on_the_isolated_network": "Keep it in a safe place and do not share or store mnemonics in any shared or non-secure environment, such as emails, photo albums, social network, message, etc.",
```
Comment (decentraliser) [inline src/language/en-US.json]: Messages should be in plural form

File: src/language/en-US.json [hunk 3]
```diff
@@ -342,58 +343,58 @@
...
   "click": "Click",
-  "Skip_this_action_but_please_confirm_your_risk_If_you_need_to_back_up_the_mnemonic_again_you_can_find_it_in_the_Wallet_Details_Export_mnemonic": "To skip this action, but please confirm your risk. If you need to back up the mnemonic again, you can find it in the Wallet Details - Export mnemonic.",
+  "Skip_this_action_but_please_confirm_your_risk_If_you_need_to_back_up_the_mnemonic_again_you_can_find_it_in_the_Wallet_Details_Export_mnemonic": "To skip this action confirm to acknowledge the risk you are taking. If you need to back up the mnemonic later or again you can find it in the 'Wallet Details - Export mnemonic'.",
```
Comment (decentraliser) [inline src/language/en-US.json]: Find it in ~the~

File: src/language/en-US.json [hunk 4]
```diff
@@ -445,33 +446,33 @@
...
   "New_root_space": "New root space",
   "Subspace": "Sub-namespace",
-  "Input_space_name": "Input space name",
-  "namespace_tips_key_1": "1. The length of the root namespace is limited to 16 characters, and the length of the secondary namespace is limited to 64 characters. Valid characters:",
-  "namespace_tips_key_2": "2. The first character of the namespace must use English letters. The namespaces that are not allowed include strings:",
-  "namespace_tips_key_3": "3. The namespace can be defined as up to three layers.",
+  "Input_space_name": "Input namespace",
+  "namespace_tips_key_1": "1. The length of the root namespace is limited to 16 characters, the length of the secondary namespace is limited to 64 characters. Valid characters:",
+  "namespace_tips_key_2": "2. The first character of the namespace must use English letters. Values not allowed include:",
+  "namespace_tips_key_3": "3. The namespace can be defined as up to three levels deep.",
```
Comment (decentraliser) [inline src/language/en-US.json]: Should be variable

File: src/language/en-US.json [hunk 5]
```diff
@@ -445,33 +446,33 @@
...
   "namespace_tips_value_1": "a, b, c, ..., z, 0, 1, 2, ..., 9, _ , -",
   "namespace_tips_value_2": "nem, user, account, org, com, biz, net, edu, mil, gov , info.",
   "undefined": "Undefined",
-  "namespace_duration_tip_1": "The duration is calculated in blocks, one block is 12 seconds, one block = 1XEM, and the maximum validity period is 365 days.",
+  "namespace_duration_tip_1": "The duration is calculated in blocks, one block is 15 seconds, one block = 1XEM, and the maximum period is 365 days.",
```
Comment (decentraliser) [inline src/language/en-US.json]: Should be variable

File: src/language/en-US.json [hunk 6]
```diff
@@ -445,33 +446,33 @@
...
   "rent": "Rent",
-  "Namespace_and_Sub_Namespace": "Namespace and Sub Namespace",
+  "Namespace_and_Sub_Namespace": "Namespace and Sub-namespace",
   "Update_namespace_prompts": "Update namespace prompts",
-  "namespace_list_tips_1": "1. Only the root namespace needs to be updated. All subspace namespaces will be automatically expanded after the root namespace is expanded;",
-  "namespace_list_tips_2": "2. According to the lease agreement, the contract with the namespace can be up to one year, and the contract can be renewed one month before or after its due date. If the namespace is not expanded in time, the name created under it and All subspaces of the mosaic will be lost;",
-  "this_account_is_already_converted": "This account has already been converted to a multisig account",
+  "namespace_list_tips_1": "1. Only the root namespace needs to be updated. All subs-namespaces will be automatically renewed with the root namespace;",
+  "namespace_list_tips_2": "2. Namespaces can be registered for up to 10 years and you have a grace period of 30 days after expiration to renew. If the namespace expires the root namespace and any of the sub-namespaces under it will be lost;",
```
Comment (decentraliser) [inline src/language/en-US.json]: Variables
Dot instead of semi at sentence end

File: src/language/en-US.json (hunk 4 above)
Comment (kaiyzen) [reply src/language/en-US.json]: which is "variable" of the three lines?

Comment (decentraliser) [reply src/language/en-US.json]: Max namespace level, max namespace length, etc...

Comment (decentraliser) [reply src/language/en-US.json]: I can fix that later

File: src/language/en-US.json (hunk 6 above)
Comment (kaiyzen) [reply src/language/en-US.json]: any ; was left there, not sure where they all appear in UI

Comment (decentraliser) [reply src/language/en-US.json]: They will appear (if the key is used, ofc)

**Pull request merged**